tests/python/gaia-ui-tests/gaiatest/tests/functional/ftu/test_ftu_skip_tour_desktopb2g.py

```diff
@@ -42,9 +43,9 @@ def test_ftu_skip_tour_for_desktop(self):
 
         self.ftu.tap_next_to_welcome_browser_section()
 
-        # Tap the statistics box and check that it sets a setting
+        # Tap the statistics box and check that it unsets this setting
```

Comment:
I don't see a lot of value in updating this comment. What the next 2 lines do are explicit enough. I would remove the comment.